pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -379,9 +390,13 @@ func (ir *IngressReconciler) Reconcile(isvc *v1beta1.InferenceService) error {
 				path = constants.PredictPath(isvc.Name, constants.ProtocolV2)
 			}
 		}
+		hostPrefix := isvc.Name
+		if disableIstioVirtualHost == true {
+			hostPrefix = fmt.Sprintf("%s-%s-default", hostPrefix, string(constants.Predictor))
```

Comment:
For transformer case it uses the transformer ksvc host name

Reply:
@yuzisun updated!